compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -1545,27 +1562,7 @@ private void validateActionInvocation(DiagnosticPos pos, BLangInvocation iExpr)
     }
 
     private void validateActions(DiagnosticPos pos, BLangNode bLangNode) {
```

Comment:
We really don't need this method. :) We can use validateActionParentNode directly.